lib/matplotlib/text.py

```diff
@@ -1044,6 +1045,34 @@ def set_fontsize(self, fontsize):
         self._fontproperties.set_size(fontsize)
         self.stale = True
 
+    def get_math_fontfamily(self):
+        """
+        Return the font family name for math text rendered by Matplotlib.
+        The default value is :rc:`mathtext.fontset`.
+        See Also
+        --------
+        set_math_fontfamily
+        """
+        return self._fontproperties.get_math_fontfamily()
+
+    def set_math_fontfamily(self, fontset):
```

Comment:
As above
```suggestion
    def set_math_fontfamily(self, fontfamily):
```

(or `family`).